core/src/main/scala/org/apache/spark/metrics/MetricsSystem.scala

```diff
@@ -20,6 +20,8 @@ package org.apache.spark.metrics
 import java.util.Properties
 import java.util.concurrent.TimeUnit
 
+import org.eclipse.jetty.servlet.ServletContextHandler
```

Comment:
Import ordering nit: I think this should be grouped with the `org.codahale.*` group a few lines down.